src/plugins/inputs/msg.js

```diff
@@ -19,7 +20,12 @@ module.exports = function(network, chan, cmd, args) {
 		target = chan.name;
 	}
 	var msg = args.join(" ");
-	irc.send(target, msg);
+	var otrSession = client.otrStore.getSession(chan.name, network);
+	if (otrSession && (otrSession.msgstate == OTR.CONST.MSGSTATE_ENCRYPTED)) {
```

Comment:
Does `otrSession.msgstate` contains the state of the last message? A bit too stateful IMO :P
If there is a stateless way of doing so, could you do something like `this.otrStore.isEncrypted(msg)` that would encapsulate these 2 checks and prevent you to require `OTR` line 2 and retrieve the `otrSession` line 23?